EgammaAnalysis/interface/LongDeps.h

```diff
@@ -14,6 +14,7 @@ class LongDeps{
         inline float energyFH() const {return energyFH_;}
         inline float energyBH() const {return energyBH_;}
         inline const std::vector<float>& energyPerLayer() const {return energyPerLayer_;}
+        inline float energy(size_t layer) const {return (layer<layers_.size()) ? energyPerLayer_[layer] : 0.;}
```

Comment:
Why this extra member? `energyPerLayer_` is initialized with 0s for all layers, and the layers.size check is incorrect: e.g. the cluster can contain layer 30 while spanning only 20 layers.

Reply:
The function signature in VARIABLES.md was something like this, so I assumed it was just missing. By all means please fix this as necessary, I was just guessing.

Reply:
I guess it should be `energyPerLayer_.size()`